components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -1875,8 +1881,15 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                         }
                         fields.put(subParts[0].replace(":", "_"), value);
                     } else {
-                        String value = subParts[1];
-                        if(value.contains(" ")) {
+                        String value = subParts[1].toLowerCase();
+
+                        if(value.contains(" or ")){
```

Comment:
The way you use String literals in above method is bit fragile, please use proper constants.